src/interactive-window/editor-integration/codeGenerator.ts

```diff
@@ -197,32 +161,7 @@ export class CellHashProvider implements ICellHashProvider {
             };
 
             traceInfo(`Adding hash for ${expectedCount} = ${hash.hash} with ${stripped.length} lines`);
-
-            let list = this.hashes.get(uristring);
-            if (!list) {
-                list = [];
-            }
-
-            // Figure out where to put the item in the list
-            let inserted = false;
-            for (let i = 0; i < list.length && !inserted; i += 1) {
-                const pos = list[i];
-                if (hash.line >= pos.line && hash.line <= pos.endLine) {
-                    // Stick right here. This is either the same cell or a cell that overwrote where
-                    // we were.
-                    list.splice(i, 1, hash);
-                    inserted = true;
-                } else if (pos.line > hash.line) {
-                    // This item comes just after the cell we're inserting.
-                    list.splice(i, 0, hash);
-                    inserted = true;
-                }
-            }
-            if (!inserted) {
-                list.push(hash);
-            }
-            this.hashes.set(uristring, list);
-
+            this.storage.store(cell, hash);
             // Save a regex to find this file later when looking for
             // exceptions in output. Track backs only work on local files.
             if (!this.traceBackRegexes.has(uristring)) {
```

Comment:
Will be removing these properties, can go into the `tracebackFormatter` we can generate the regex on the fly after some some string lookups.